src/main/java/frc/robot/Robot.java

```diff
@@ -121,23 +187,29 @@ public void teleopPeriodic()
     }
     // barMotor.fsset(Math.pow(joystick.getRawAxis(4), 3));
     SmartDashboard.putNumber("bar enc", barMotor.getEncoder().getPosition());
+
+    // Emergency dump button in case the colour sensor senses the wrong colour
+    if (joystick.getRawButton(8)) {
+      ballIsBlue = false;
+      ballIsRed = false;
+      dumpBall = true;
+    }
   }
 
   @Override
-  public void disabledInit() {
-    hookMotor.setIdleMode(IdleMode.kCoast);
-  }
+  public void disabledInit() {}
```

Comment:
I want to make sure that the changes to the methods `disabledInit()` and `disabledPeriodic()` are intentional? I remember @newjanson was making changes here, so I want to make sure that this is okay.

Reply:
That should probably remain as it was before, its for the bar PID

Reply:
Yeah, he set it to coast mode on saturday so it's part of the original code

Reply:
So the changes in the PR are good?

Reply:
Disabled init should have hookMotor.setIdleMode(IdleMode.kCoast)

Reply:
Can you make the changes for that @WhyNot180? And can you also double check `disabledPeriodic()` to make sure that change was intentional?

Reply:
Okay, I fixed it and pushed it

Reply:
Thank you! I'm going to wait for Greg to review it before merging